source/views/settings/debug/list.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react'
+import {FlatList} from 'react-native'
+import {DebugRow} from './row'
+import {connect} from 'react-redux'
+import {NoticeView} from '../../../components/notice'
+import {ListSeparator} from '../../../components/list'
+import toPairs from 'lodash/toPairs'
+import {toLaxTitleCase} from 'titlecase'
+import get from 'lodash/get'
+import type {NavigationState} from 'react-navigation'
+import type {TopLevelViewPropsType} from '../../types'
+
+type Props = TopLevelViewPropsType & {
+	state: any,
+}
+
+export class DebugListView extends React.PureComponent<Props> {
+	static navigationOptions = ({navigation}: NavigationState) => {
+		let titleParam = navigation.getParam('keyPath', 'Debug')
```

Comment:
It was a very convenient way to fallback onto the top-level name if we were not n+1 levels deep in the stack. I'm open to suggestions for here or other places.

Reply:
I'm asking why you aren't using getParam instead of accessing `navigation.state.params.keyPath` in the rest of the code, but you are using it here.

I think that you should use it in the rest of the code.

Reply:
That makes sense. I'll make that happen.